docs: refine the user guide (#1696)
GreptimeTeam/docs: docs/user-guide/overview.md

---
keywords: [metrics, logs, observability, SQL support, range queries, flow computation, data model, protocols]
description: Learn how to use GreptimeDB according to your use case, including data ingestion, querying, and administration.
---

# User Guide

Welcome to the user guide for GreptimeDB.

> GreptimeDB is the unified time series database for metrics, logs, and traces,

Reviewer: ```suggestion
GreptimeDB is the unified observability database for metrics, logs, and traces,
```

providing real-time insights from Edge to Cloud at any scale.

## Understanding GreptimeDB Concepts

Before diving into GreptimeDB,
it is recommended to familiarize yourself with its data model, key concepts, and features.
For a comprehensive overview, 
refer to the [Concepts Documentation](./concepts/overview.md).

## Ingesting Data Based on Your Use Case

GreptimeDB supports [multiple protocols](./protocols/overview.md) and [various integration tools](./integrations/overview.md) to simplify data ingestion tailored to your requirements.

> ### For Observability Metrics

Reviewer 2: Combine this with logs, and add traces too. Rename the section as "For Observability Data"

> ### For Observability Metrics

Reviewer: ```suggestion
### For Observability Scenarios
```

> If you plan to use GreptimeDB as metrics storage for observability purposes,

Reviewer: ```suggestion
If you plan to use GreptimeDB as metrics, logs and traces storage for observability purposes,
```

see the [Observability Documentation](./ingest-data/for-observability/overview.md).

> It explains how to ingest data using tools like Vector, Kafka, Prometheus, and the InfluxDB line protocol.

Reviewer: ```suggestion
It explains how to ingest data using tools like Otel-Collector, Vector, Kafka, Prometheus, and the InfluxDB line protocol.
```

> ### For IoT and Edge Computing

Reviewer: ```suggestion
### For IoT and Edge Computing Scenarios
```

For IoT and Edge Computing scenarios,
the [IoT Documentation](./ingest-data/for-iot/overview.md) provides comprehensive guidance on ingesting data from diverse sources.

> ### For Log Storage

Reviewer: Could be combind with the `For Observability Scenarios`.

To use GreptimeDB as a log storage solution,
refer to the [Logs Documentation](./logs/overview.md).
It details how to ingest pattern text logs using pipelines.

### Accelerating Queries with Indexes

Indexes such as inverted indexes, skipping indexes, and full-text indexes can significantly enhance query performance.
Learn more about effectively using these indexes in the [Data Index Documentation](./manage-data/data-index.md).

> ## Querying Data for Insights

Reviewer: This section must be above `Accelerating Queries with Indexes`.

You should learn how to query data, then cccelerating them.
